Lux/src/next_element.hpp

```diff
@@ -536,4 +536,34 @@ struct next_element {
     next_element();
 };
 
+// Audio-reactive function types integrated with Joy's harness system
+// These work directly with the scene's global context instead of separate audio context
+
+struct audio_float_fn {
+    std::string channel;
+    float sensitivity;
+    float base_value;
```

Comment:
What is the purpose of base value here? Do you use it to remember the original value of the slider?